ext/stringio/stringio.c

```diff
@@ -629,7 +629,14 @@ strio_closed(VALUE self)
  * call-seq:
  *   closed_read? -> true or false
  *
- * Returns +true+ if +self+ is closed for reading, +false+ otherwise.
+ * Returns whether +self+ is closed for reading:
+ *
+ *   strio = StringIO.new
+ *   strio.closed_read?   # => false
+ *   strio.close_read
+ *   strio.closed_read?   # => true
+ *
+ * Related: StringIO.closed?, StringIO.closed_write?.
```

Comment:
Could you also add `close_read`?

BTW, should we use `#` not `.` for separator such as `StringIO#closed?`?

Reply:
Fixed.